apps/backoffice/src/components/services/service-context-menu.tsx

```diff
@@ -210,6 +213,20 @@ export const ServiceContextMenu = ({
     </Tooltip>
   );
 
+  /** Show App Preview action button */
+  const renderAppPreviewAction = () => (
```

Comment:
Refactor tip: we have here 2 very similar buttons inside function `renderHistoryAction` and here `renderAppPreviewAction`. I suggest to refactor this shared code by creating a shared "IconButtonWithTooltip" inside components/buttons and use it instead duplicate the same button 2 times.

Reply:
[00f977d](https://github.com/pagopa/io-services-cms/pull/810/commits/00f977dd8490a63b3e71ad934b920baeadfd6d85)